tests/functional/run-s3-tests.sh

```diff
@@ -10,14 +10,26 @@ run_sds || exit 1
 configure_aws
 configure_s3cmd
 
+CH_VERSIONING=$(python -c "import oio, sys; print(oio.__version__ > '4.5.1')")
+
 RET=0
 
-run_functional_test s3-container-hierarchy.cfg s3_container_hierarchy_v2.sh s3-marker.sh
-run_functional_test s3-container-hierarchy-key-v2.cfg s3_container_hierarchy_v2.sh s3-marker.sh
+run_functional_test s3-container-hierarchy.cfg s3_container_hierarchy_v2.sh s3-marker.sh s3-versioning.sh
+# run only CH versioning
+if [ "${CH_VERSIONING}" == "True" ]; then
+    run_functional_test s3-container-hierarchy.cfg s3-versioning.sh
```

Comment:
The pipeline of this configuration doesn't contain `versioned_writes`. We should create a new configuration file.

Reply:
`s3-versioning.sh` create files only at the root. We should create a new script for CH with versioning.

Reply:
My bad !